### ISSUE TITLE
CIDR notation is not applied for bigip_ltm_virtual_server

### ISSUE THREAD

**Issue by slawekww** [F5Networks/terraform-provider-bigip#736]
### Environment

 * TMOS/Bigip Version:  BIG-IP 16.1.3.2 Build 0.0.4 Point Release 2
 * Terraform Version: v1.3.5
 * Terraform bigip provider Version:  1.16.0

### Summary

Once bigip_ltm_virtual_server is created or updated, **mask** parameter works only for decimal values.
If mask is provided as CIDR notation, it does not apply on destination address.

### Steps To Reproduce

1. Provide terraform resource config which you are facing trouble along with the output of it:

TF code:
```
resource "bigip_ltm_virtual_server" "http-k8s" {
  name                       = "/f5part/vs-http"
  destination              = "10.0.0.2"
  mask                        = "255.255.255.254"
  ip_protocol              = "tcp"
  description              = "Virtual server"
  port                         = 80
  source_address_translation = "automap"
}
```

3. To get to know more about the issue, provide terraform debug logs

Result:
```
  # module.f5.bigip_ltm_virtual_server.http-k8s will be updated in-place
  ~ resource "bigip_ltm_virtual_server" "http-k8s" {
        id                         = "/f5part/vs-http"
        destination          = "10.0.0.2"
        mask                    = "255.255.255.254"
        name                   = "/f5part/vs-http"
```

However mask is set to 255.255.255.255.

Once mask is change to decimal **31** change is recognized:

```
 # module.f5.bigip_ltm_virtual_server.http-k8s will be updated in-place
  ~ resource "bigip_ltm_virtual_server" "http-k8s" {
        id                         = "/f5part/vs-http"
      ~ mask                       = "255.255.255.255" -> "31"
        name                       = "/f5part/vs-http"
        # (15 unchanged attributes hidden)
    }

```

### Expected Behavior
CIDR notation should work for mask.

### Actual Behavior
CIDR notation is ignored and apply 255.255.255.255 always.

**Comment by slawekww** [comment]
For each Terraform plan, it recognizes change `mask                       = "255.255.255.254" -> "31"` even if plan contains decimal value.

**Comment by RavinderReddyF5** [comment]
Hi @slawekww fixed in v1.16.1, please open new issue if you see any problem
